mongodb/Acornfile

```diff
@@ -0,0 +1,331 @@
+args: {
+    // Specify whether MongoDB instance is a standalone or replica set
+    isReplicaSet: false
+    // Custom user to be created during the initialization
+    dbUserName: ""
+    // Custom database to be created during the initialization
+    dbName: "acorn"
+    replicas: 1
+    // Enable diagnostic mode
+    diagnosticMode: false
+    // Enable authentication
+    authEnabled: true
+    // Run arbiter node
+    // Mongodb additional command line flags
+    extraFlags: ""
+    arbiterEnabled: false
+    // Arbiter additional command line flags
+    arbiterExtraFlargs: ""
+    // Run hidden node
+    hiddenEnabled: false

```

Comment:
Can this be collapsed into a single arg? `--hiddenReplicas=0` and when > 0 it is enabled?

Reply:
ok